environments/development/public.yml

```diff
@@ -44,6 +44,7 @@ nofile_limit: 65536
 
 supervisor_http_enabled: True
 
+couchdb_use_haproxy: True
```

Comment:
how long do you expect we'll be running nginx & haproxy?

Reply:
as long as it takes me to roll this out everywhere

Reply:
but hopefully not longer than a month